bot/cogs/moderation/scheduler.py

```diff
@@ -266,9 +305,9 @@ def mod_log(self) -> ModLog:
         mod_role = guild.get_role(constants.Roles.moderator)
         user_id = infraction["user"]
         _type = infraction["type"]
```

Comment:
I know you didn't touch this but while we're fixing the `id_` should this be `type_`?

Reply:
Yeah I went ahead and fixed some of the other ones in the code base (using the regex `^\s*?_\w+`).